lib/std/debug/MemoryAccessor.zig

```diff
@@ -38,8 +38,8 @@ pub fn deinit(ma: *MemoryAccessor) void {
 
 fn read(ma: *MemoryAccessor, address: usize, buf: []u8) bool {
     switch (native_os) {
```

Comment:
You could also try deleting this switch entirely and seeing if CI still passes.  I just did a preliminary test which suggests that the issue this was working around is no longer present.

Reply:
@jacobly0 I think is what you had in mind? 

I didn't scan through the commit history to understand if this would be the simplest revert, and also because I don't have the context around the struct, I'm very much depending on CI + review here to validate this change. Let me know if I missed something obvious?